src/components/TextInput/styles.tsx

```diff
@@ -0,0 +1,55 @@
+import styled from 'styled-components';
+import { Inter } from 'next/font/google';
+import COLORS from '../../styles/colors';
+
+const inter = Inter({ subsets: ['latin'] });
+
+export const InputText = styled.input<{ $error: boolean }>`
+  color: ${COLORS.existingInputColor};
+  font-size: 1rem; // 16px
+  ${inter.style}
+  font-weight: 400;
+  line-height: 19.2px;
+  word-wrap: break-word;
+  padding: 0.625rem 1.25rem; // 10px 20px 10px 20px;
+  border-radius: 5px;
+  border: 2px solid
+    ${({ $error }) =>
+      $error ? COLORS.errorColor : COLORS.inputSuggestionColor};
+  justify-content: flex-start;
+  align-items: flex-start;
+  gap: 10px;
+  color: ${COLORS.existingInputColor};
+
+  &:focus {
+    outline: none !important;
+    border: 2px solid
+      ${({ $error }) => ($error ? COLORS.errorColor : COLORS.activeGreen)};
+  }
+  &::placeholder {
```

Comment:
VERY MINOR: maybe an extra line break before this line 🍵

Reply:
will do!